third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -547,6 +599,28 @@ xla::PjRtDevice* PjRtComputationClient::StringToPjRtDevice(
   return pjrt_device;
 }
 
+void PjRtComputationClient::WaitDeviceExections(
+    const std::vector<std::string>& devices) {
+  std::set<std::string> wait_devices;
```

Comment:
Do we need an ordered set here, a rb tree?

Reply:
yea unordered_set works better.

Reply:
What is the set for? Would it be better to just directly iterate through `devices` if not empty and `GetLocalDevices` otherwise?

Reply:
it is in case people passed in duplicated device names, I should make it a unordered_set.